packages/components-native/src/ContentOverlay/ContentOverlay.style.ts

```diff
@@ -54,18 +54,18 @@ export const styles = StyleSheet.create({
     opacity: 0,
   },
 
-  titleWithoutDimiss: {
+  titleWithoutDismiss: {
     flex: 1,
     flexDirection: "row",
     justifyContent: "center",
     paddingTop: tokens["space-base"],
+    paddingHorizontal: tokens["space-base"],
   },
 
   titleWithDismiss: {
     flex: 1,
     justifyContent: "center",
     paddingLeft: tokens["space-base"],
     paddingRight: tokens["space-smaller"],
-    flexGrow: 1,
```

Comment:
`flex: 1` is already `flexGrow: 1` and `flexShrink: 1` adding one more is not only redundant but also makes the styling unpredictable as iOS seems to reset back the `flexShrink` to `0` if you specify `flexGrow`